internal/cmd/server.go

```diff
@@ -84,7 +84,7 @@ func newServerCmd() *cobra.Command {
 	cmd.Flags().Duration("session-duration", 0, "Maximum session duration per user login")
 	cmd.Flags().Duration("session-extension-deadline", 0, "A user must interact with Infra at least once within this amount of time for their session to remain valid")
 	cmd.Flags().Bool("enable-signup", false, "Enable one-time admin signup")
-	cmd.Flags().String("hostname", "example.com", "hostname for the server")
+	cmd.Flags().String("hostname", "", "hostname for the server")
```

Comment:
I don't think we should be adding more flags for `infra server`. We should use env vars or config files for setting server options. Command line flags work well for interactive commands run by humans. Env vars and config files work well for long running processes that are generally run by other processes.

This can be set with `INFRA_SERVER_HOSTNAME`

Reply:
my only concern there is it's pretty much a required field, as saas won't function without it. env variables always seem more like optional things to me.